spring-cloud-alibaba-starters/spring-cloud-alibaba-sentinel-datasource/pom.xml

```diff
@@ -81,6 +85,12 @@
             <optional>true</optional>
         </dependency>
 
+        <dependency>
+            <groupId>com.alibaba.csp</groupId>
+            <artifactId>sentinel-datasource-opensergo</artifactId>
+            <version>${opensergo-datasource.version}</version>
+        </dependency>
```

Comment:
Whether to use` optional`, the integration of `opensergo` is at the discretion of the user.

Reply:
Without this dependency, I can not use the OpenSergoDataSourceGroup